model/ftn/wmesmfmd.ftn

```diff
@@ -2826,6 +3049,44 @@
         enddo
       endif
 !
+! 2.h Set ESMF import grid corner coordinates, only for coupled with mediator
+!
+      if (med_present) then 
```

Comment:
can this be for even if a mediator is not present?

Reply:
I think we should see if we can remove the "med_present" for this to see if it still works.  I can help test this in the global tests to see if this causes changes/issues if needed.  If we can't test this before this PR is merged we should make an issue to revisit this.

Reply:
This part of code is working without the check. So, I am removing "med_present" from the part of the code that includes the corner coordinates.

Reply:
It would be nice to check it with global application.